catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/searchui/standard/endpoints/WorkspaceBodyWriter.java

```diff
@@ -46,8 +50,8 @@ public void writeTo(
       MediaType mediaType,
       MultivaluedMap<String, Object> httpHeaders,
       OutputStream entityStream)
-      throws IOException, WebApplicationException {
+      throws IOException {
 
-    JsonFactory.create().writeValue(entityStream, o);
+    new PrintWriter(new OutputStreamWriter(entityStream)).print(GSON.toJson(o));
```

Comment:
❓ How about using `GSON.toJson(o, new OutputStreamWriter(entityStream))`? 
We should also make sure to flush the writer before leaving?

Reply:
Much better. Thanks.